packages/fuchsia_ctl/bin/main.dart

```diff
@@ -64,9 +69,8 @@ Future<void> main(List<String> args) async {
             'publish, or serve.')
     ..addCommand('serve')
     ..addCommand('newRepo');
-  pmSubCommand
-      .addCommand('publishRepo')
-      .addMultiOption('far', abbr: 'f', help: 'The .far files to publish.');
+  pmSubCommand.addCommand('publishRepo')
+    ..addMultiOption('far', abbr: 'f', help: 'The .far files to publish.');
```

Comment:
nit - why change this?

Reply:
I added a second option for help command that I removed. Getting it back to "."